src/index.js

```diff
@@ -23,7 +23,7 @@ const agenda = new Agenda({
   }
 });
 
-agenda.on('ready', () => {
+const agendaReady = new Promise(resolve => agenda.on('ready', () => {
```

Comment:
👍 👍 👍

Reply:
Note that this might go in at some point: https://github.com/agenda/agenda/pull/557